package.json

```diff
@@ -73,6 +73,7 @@
     "chokidar-cli": "^1.2.2",
     "common-tags": "^1.8.0",
     "dotenv": "^8.2.0",
+    "eleventy-plugin-toc": "github:MichaelSolati/eleventy-plugin-toc#feat/options",
```

Comment:
So a few things my fork did was 1) allow for us to apply options for the plugin when we configure it with 11ty. The inline options just would not work. Also I added options so we can use an `ul` instead of an `ol` as well as allow for a flat list, rather than embedded `ul`s.

Reply:
I guess we need to discuss what we should do with the library, if we want to continue with it, etc... Chat on Monday?

Reply:
Can you copy this into a directory in the _filters dir?